### PR TITLE
Blog

### PR DESCRIPTION
**Pull request by alan2000alex**
All blog stuff included:
template,
content : blog listing,
sass : blog>main.scss

### CODE REVIEW

File: content/blog/blog1.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
++++
+title = "Blog 1"
+date = 2021-08-24
+description = "Idli World"
```
Comment (realaravinth) [inline content/blog/blog1.md:4]: @alan2000alex seriously? :joy:

File: sass/blog/main.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
+@import '../_vars';
+
+.page__link-list {
+    margin-top: 1em;
```
Comment (realaravinth) [inline sass/blog/main.scss:4]: Please use `rem` over `em`. They scale well when the user has set a larger font size

File: sass/blog/main.scss [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+.link-list__item >h2 {
+    font-size: 1.4em;
+    color: $dark-black;
+}
+
+.link-list__item>h2:hover{
+    color: $deep-red;
+    // transition-delay: 20ms;
+    transition-duration: .3s;
```
Comment (realaravinth) [inline sass/blog/main.scss:24]: Store transition  in a variable and reuse it where ever required

**Review by realaravinth — CHANGES_REQUESTED** [review]
Good work overall but please get rid of the placeholder blogs and publish the stuff that Dhruva wrote

File: content/blog/blog1.md (hunk 1 above)
Comment (alan2000alex) [reply content/blog/blog1.md:4]: apologies xD

File: sass/blog/main.scss (hunk 2 above)
Comment (alan2000alex) [reply sass/blog/main.scss:4]: Changes made.

File: sass/blog/main.scss (hunk 3 above)
Comment (alan2000alex) [reply sass/blog/main.scss:24]: Changes made.

**Pull request merged**